CMakeLists.txt

```diff
@@ -142,23 +142,24 @@ option(NGRAPH_DEX_ONLY "Build CPU DEX without codegen" FALSE)
 option(NGRAPH_CODE_COVERAGE_ENABLE "Enable code coverage data collection" FALSE)
 option(NGRAPH_LIB_VERSIONING_ENABLE "Enable shared library versioning" FALSE)
 option(NGRAPH_PYTHON_BUILD_ENABLE "Enable build nGraph python package wheel" FALSE)
-option(NGRAPH_DISTRIBUTED_MLSL_ENABLE "Add distributed MLSL mode for CPU only backend" FALSE)
-option(NGRAPH_DISTRIBUTED_OMPI_ENABLE "Add distributed Open-MPI mode for all backend" FALSE)
 option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ${PLAIDML_FOUND})
+option(NGRAPH_DISTRIBUTED_ENABLE "Enable distributed training using MLSL/OpenMPI" OFF)
 
-if (NGRAPH_GPUH_ENABLE)
-    set(NGRAPH_GPU_ENABLE TRUE)
+if (NGRAPH_DISTRIBUTED_ENABLE)
+    if ("${NGRAPH_DISTRIBUTED_ENABLE}" STREQUAL "MLSL")
+        set(NGRAPH_DISTRIBUTED_MLSL_ENABLE TRUE)
+    elseif("${NGRAPH_DISTRIBUTED_ENABLE}" STREQUAL  "OMPI")
+        set(NGRAPH_DISTRIBUTED_OMPI_ENABLE TRUE)
+    else()
+        message(FATAL_ERROR
+                    "Invalid arguments passed to NGRAPH_DISTRIBUTED_ENABLE .\n"
+                    "If CPU only backend recommend Intel MLSL by setting -DNGRAPH_DISTRIBUTED_ENABLE=MLSL .\n"
```

Comment:
Unless you want to fully document when the options are used just day "must select one of MLSL, OMPI or OFF